lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+
+The Libra Association is an independent membership organization that is responsible for the governance of the LPN and the development of the Libra project. The Association is governed by the Association Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board.
+
+Each member of the Libra Association is expected to be ready to run a replica of the Libra protocol, called a Validator Node. Validator Nodes coordinate and reach consensus on the state of the blockchain by processing transactions, they ensure correct updates even if some Validator Nodes (fewer than a third) fail in arbitrary ways. Though only a subset of members, called a Validator Set, run Validator Nodes in any given time. The Association determines the composition of the Validator Set by adding and removing members over time. Reasons for changing the Validator Set include new members joining the Association or existing members leaving it and actions needed to comply with LN’s obligations under its FINMA License designed to guarantee the secure and performant operation of the platform. The Validator Set is stored as part of the blockchain state, in an on-chain configuration. 
+
+# Terminology
+* *Validator Owner* is an owner of the Libra account of the Libra Association's member. The account stores the *Validator Config* resource.
+* *Validator Config* is an on-chain resource that contains information for running a Validator Node. It includes a *consensus public key*, which is an EdDSA public key used to authenticate consensus messages. It also includes *network information*: network addresses and static public keys for establishing secure connections with this Validator Node from other Validator Nodes or from clients.
+* *Validator Set* is an on-chain resource that stores a vector, where each entry contains an address of a Validator Owner and a copy of its Validator Config. Though the Validator Config in the Validator Set may be an older version of the Validator Config currently stored under Validator Owner's account, each Validator Node is run with the Validator Config from the Validator Set.
+Each address appears at most once in the vector. The Libra Association collectively decides which Validator Owners' addresses are in the Validator Set. There is a one-to-one correspondence between a Validator Owner in the Validator Set and a Validator Node. The Validator Set changes, either when one of the Validator Nodes updates their parameters or when a new member is added or removed, the Reconfiguration happens.
```

Comment:
We should probably re-read and unify validator config and validator node where appropriate and be clear that a validator config defines either the current or potential configuration for a validator node.